src/applications/lgy/coe/tests/config/decoration.unit.spec.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import sinon from 'sinon';
+import { expect } from 'chai';
+import { Provider } from 'react-redux';
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```

Comment:
This should be updated to applicant service decoration

Reply:
updated